src/zobrist.rs

```diff
@@ -107,11 +107,84 @@ mod tests {
 
     #[test]
     fn full_hands() {
-        let pos = Position::new(
+        let all_black_hands = Position::new(
             PartialPosition::from_usi("sfen 9/9/9/9/9/9/9/9/9 b 2R2B4G4S4N4L18P 1")
                 .expect("failed to parse"),
         );
-        assert_ne!(0, pos.key());
+        let all_white_hands = Position::new(
+            PartialPosition::from_usi("sfen 9/9/9/9/9/9/9/9/9 b 2r2b4g4s4n4l18p 1")
+                .expect("failed to parse"),
+        );
+        assert_eq!(0, all_black_hands.keys().0);
+        assert_eq!(0, all_black_hands.keys().0);
```

Comment:
Probably a typo?
```suggestion
        assert_eq!(0, all_white_hands.keys().0);
```

Reply:
Oops ... Thank you for pointing it out 🙇